docs/section19/19-3.md

```diff
@@ -19,8 +19,9 @@
 - Get the weather for the ruleset project description: `weather = RulesetProjectDescription.weather`
 
 - **Rule Assertion:**
-- Case 1: Heating/cooling design day types do not match the definitions in 90.1-2019; outcome=FAIL: `if (weather.cooling_design_day_type != COOLING_1_0) or (weather.heating_design_day_type != HEATING_99_6): FAIL`
-- Case 2: Otherwise, conduct manual check that rule is met; outcome=UNDETERMINED:  `else: UNDETERMINED and raise_message "Check that the weather conditions used in sizing runs to determine baseline equipment capacities is based on design days developed using 99.6% heating design temperatures and 1% dry-bulb and 1% wet-bulb cooling design temperatures."`  
+- Case 1: Heating/cooling design day types do not match the definitions in 90.1-2019; outcome=FAIL: `if (weather.cooling_design_day_type != COOLING_1_0 or weather.heating_design_day_type != HEATING_99_6): FAIL`
```

Comment:
The logic of this won't work because if they equal null it will return a fail since this is the first case. I would put case 2 first, then case 1 second, and then case 3

Reply:
woops!